### PR TITLE
feat: add assets for employee brage

### PR DESCRIPTION
**Pull request by braaar**
Not sure if my github process for this is on point, but here goes...

I added info about myself to the employee section. Didn't know what to put as ID, so I gave myself 3 (0, 1 and 2 were there before).
The photo is the best I could find lying around for now, but I want to take a new picture that is less... sinister.

### CODE REVIEW

**Comment by braaar** [comment]
I would assume these checks fail due to something on the `develop` branch, and not from my commits?

File: web/src/data/en/employees/brage-sekse-aarset.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+language: en
+type: employee
+id: 3
+name: Brage Sekse Aarset
+position: Software developer
```
Comment (cobraz) [inline web/src/data/en/employees/brage-sekse-aarset.mdx:6]:
```suggestion
position: Software Developer
```

**Review by cobraz — APPROVED** [review]
Looks good! The checks failed due to some issues on `develop` branch, yes!

**Comment by cobraz** [comment]
Feel free to «Squash and merge» whenever/if the checks turns green :)

File: web/src/data/en/employees/brage-sekse-aarset.mdx [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+email: brage@bjerk.io
+phoneNumber: +47 464 65 823
+photo: brage-photo.png
```
Comment (cobraz) [inline web/src/data/en/employees/brage-sekse-aarset.mdx:9]:
```suggestion
photo: brage-image.png
```

File: web/src/data/no/employees/brage-sekse-aarset.mdx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+language: no
+type: employee
+id: 3
+name: Brage Sekse Aarset
+position: Programvareutvikler
+email: brage@bjerk.io
+phoneNumber: +47 464 65 823
+photo: brage-photo.png
```
Comment (cobraz) [inline web/src/data/no/employees/brage-sekse-aarset.mdx:9]:
```suggestion
photo: brage-image.png
```

**Review by cobraz — CHANGES_REQUESTED** [review]

**Comment by cobraz** [comment]
You can ignore the one that fails there 🙂

It's not related to this pull request; I'm creating an issue on it :)

**Review by cobraz — APPROVED** [review]

**Review by AnneMatilde — APPROVED** [review]
LGTM!

**Pull request merged**